### PR TITLE
Update devcontainer image version `typescript-node:20`

### PR DESCRIPTION
**Pull request by stamat**
## Summary

Node 18 needed for Storybook 8 to be run. Current node version is 16, therefore the image version for the container is upped to 18. I just wanted to be able to run `npm run start` right away after Codespaces loads.

## List of notable changes:

- **updated** dev container image version from `typescript-node:16` to `typescript-node:20`

## What should reviewers focus on?

- Verify that the `node -v` is indeed 20

## Steps to test:

1. Create a new codespace
1. Run `node -v`
1. Should say `v20.x.x`

P.S. Tested it myself works.

## Supporting resources (related issues, external links, etc):

-
-

## Contributor checklist:

- [x] All new and existing CI checks pass
- [x] Tests prove that the feature works and covers both happy and unhappy paths
- [x] Any drop in coverage, breaking changes or regressions have been documented above
- [ ] UI Changes contain new visual snapshots (generated by adding `update snapshots` label to the PR)
- [x] All developer debugging and non-functional logging has been removed
- [x] Related issues have been referenced in the PR description

## Reviewer checklist:

- [ ] Check that pull request and proposed changes adhere to our [contribution guidelines](../../CONTRIBUTING.md) and [code of conduct](../../CODE_OF_CONDUCT.md)
- [ ] Check that tests prove the feature works and covers both happy and unhappy paths
- [ ] Check that there aren't other open Pull Requests for the same update/change

## Screenshots:

> Please try to provide before and after screenshots or videos

<table>
<tr>
<th> Before</th> <th> After </th>
</tr>
<tr>
<td valign="top">

 </td>
<td valign="top">

</td>
</tr>
</table>

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "Primer Brand",
-    "image": "mcr.microsoft.com/vscode/devcontainers/typescript-node:16",
+    "image": "mcr.microsoft.com/vscode/devcontainers/typescript-node:18",
```
Comment (rezrah) [inline .devcontainer/devcontainer.json:3]: Thanks @stamat, can we bump this to Node 20 at a minimum given 22 is now LTS?

Comment (rezrah) [reply .devcontainer/devcontainer.json:3]:
```suggestion
    "image": "mcr.microsoft.com/vscode/devcontainers/typescript-node:20",
```

Comment (stamat) [reply .devcontainer/devcontainer.json:3]: @rezrah I had it first at 20 lol, but I was like maybe it's too much of an upgrade :D Yeah I agree legend, let's go with 20!

**Review by rezrah — APPROVED** [review]
🚀

**Pull request merged**